### PR TITLE
add -v option for npx networks; display latest block number

### PR DESCRIPTION
**Pull request by IgorPetkovic**

### CODE REVIEW

File: src/internal/cli/commands/NetworksCommand.ts [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 export const NetworksCommand = new commander.Command("networks")
   .description("list all Tenderly supported networks")
-  .action(async () => {
+  .option("-v, --verbose", "display detailed network information")
+  .action(async (options) => {
+    const verbose = options.verbose != undefined && options.verbose == true;
+
+    const headers = ["Network ID", "Network slug"];
+    if (verbose) {
+      headers.push("Latest block");
```
Comment (Riphal) [inline src/internal/cli/commands/NetworksCommand.ts:14]: Maybe "Latest block number"?

File: src/internal/cli/commands/NetworksCommand.ts [hunk 2]
```diff
@@ -16,13 +24,32 @@ export const NetworksCommand = new commander.Command("networks")
...
+          if (verbose) {
+            const blockNumber = await TenderlyService.getLatestBlockNumber(
+              network.ethereum_network_id
+            );
+
+            return [network.ethereum_network_id, network.slug, blockNumber];
+          } else {
+            return [network.ethereum_network_id, network.slug];
+          }
+        })
```
Comment (Riphal) [inline src/internal/cli/commands/NetworksCommand.ts:39]: Like it 😎

**Review by Riphal — APPROVED** [review]
LGTM just one question can we change the color of the table header because now it is red which made me think something is wrong.

Screenshot 2022-07-21 at 12 03 50

File: src/internal/cli/commands/NetworksCommand.ts (hunk 2 above)
Comment (IgorPetkovic) [reply src/internal/cli/commands/NetworksCommand.ts:39]: Props to @zdenkolini for help with that 😎

Comment (zdenkolini) [reply src/internal/cli/commands/NetworksCommand.ts:40]: thanks for the praise 🤗

**Pull request merged**